src/ext/config.js

```diff
@@ -107,6 +108,7 @@ function PPConfigs() {
     cfConfig["issue-action"]["tokens-per-request"] = 30;
     cfConfig.cookies["clearance-cookie"] = "cf_clearance";
     cfConfig["captcha-domain"] = "captcha.website";
+    cfConfig["get-more-passes-url"] = "https://captcha.website";
```

Comment:
Nit: could we set `"captcha.website"` as a variable and then use that in both `cfConfig["captcha-domain"]` and `cfConfig["get-more-passes-url"]`. Alternatively, there may be a nice way of just removing the `captcha-domain` key as it is only used by CF and this seems to duplicate the usage somewhat.

Reply:
I'll give it some thought, I think `captcha-website` is also used when checking if we should redeem, so it serves another purpose

Reply:
We also use it to clear cookies on the captcha domain, so I'd leave it in.

Reply:
Okay, would it be possible to set `"captcha.website"` as a variable in this file and then just use it in both cfConfig settings?